plugins/whatsapp.js

```
@@ -4,367 +4,410 @@
-		await delay(2000);
-		await message.send('_Cleared_');
-	}
+  {
+    pattern: 'onwa',
+    public: true,
+    type: 'whatsapp',
+    desc: 'Checks if users exist on WhatsApp',
+  },
+  async (message, match) => {
+    if (!match) return message.send('_Provide their numbers, e.g. 121232343,131312424_');
+    match = match.split(',').map((id) => toJid(id.trim()));
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)